.github/workflows/dispatch-build-bottle.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       - name: Push commits
         uses: Homebrew/actions/git-try-push@master
         with:
-          token: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
+          token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
```suggestion
          token: ${{secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN}}
```

Need this to push as BrewTestBot and fire the workflow on master.

Reply:
Same in other workflows I believe

Reply:
@dawidd6 Updated.